### PR TITLE
2023 annual report for SIG scalability

### PR DESCRIPTION
**Pull request by shyamjvs**
FYI - @kubernetes/sig-scalability-leads @wojtek-t @marseel 

/cc @mrbobbytables

### CODE REVIEW

File: sig-scalability/annual-report-2023.md [hunk 1]
```diff
@@ -4,58 +4,80 @@
...
 
 3. Did you have community-wide updates in 2023 (e.g. KubeCon talks)?
 
-<!--
-  Examples include links to email, slides, or recordings.
--->
+We presented a SIG update at KubeCon Chicago 2023:
+- [SIG Scalability: Intro & Deep Dive](https://kccncna2023.sched.com/event/1R2pT/intro-deep-dive-kubernetes-sig-scalability-wojciech-tyczynski-google-marcel-zieba-isovalent)
+  - [Slides](https://static.sched.com/hosted_files/kccncna2023/85/SIG%20Scalability%20-%20Kubecon%20NA%202023.pdf)
+  - [Recording](https://youtu.be/nrAskCyG_Xk)
```
Comment (wojtek-t) [inline sig-scalability/annual-report-2023.md:24]: Also Kubecon Europe:
https://kccnceu2023.sched.com/event/1HyTL/intro-deep-dive-kubernetes-sig-scalability-wojciech-tyczynski-google

**Review by wojtek-t — COMMENTED** [review]
One minor comment - other than that LGTM

File: sig-scalability/annual-report-2023.md (hunk 1 above)
Comment (shyamjvs) [reply sig-scalability/annual-report-2023.md:24]: added, thanks

**Pull request merged**